package.json

```diff
@@ -1,48 +1,42 @@
 {
   "name": "ember-buffered-proxy",
-  "version": "0.6.0",
-  "description": "An Ember Object Proxy with change buffering",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:testall"
+    "test": "ember try:each"
   },
-  "repository": "https://github.com/yapplabs/ember-buffered-proxy",
+  "repository": "",
   "engines": {
     "node": ">= 0.10.0"
   },
-  "author": "Kris Selden, Stefan Penner & Yapp Labs",
-  "contributors": [
-    {
-      "name": "Ben Limmer",
-      "email": "hello@benlimmer.com"
-    }
-  ],
```

Comment:
Please retain the existing package.json configurations merged with dependency version updates.